autoload/cmake.vim

```diff
@@ -183,6 +183,13 @@ function! cmake#Close() abort
     call s:terminal.Close()
 endfunction
 
+" API function for :CMakeToggle
```

Comment:
Nit-picky: add a period at the end of sentence for consistency with other docstrings.

Reply:
Fixed in 6010f6df16191be561811f2db936627ad5a5cf26